src/ucp/core/ucp_mm.c

```diff
@@ -150,9 +151,8 @@ static ucs_status_t ucp_mem_alloc(ucp_context_h context, size_t length,
             }
         }
 
-        /* TODO: pass addr */
-        status = uct_mem_alloc(NULL, length, uct_flags, &method, 1, mds, num_mds,
-                               name, &mem);
+        status = uct_mem_alloc(memh->address, length, uct_flags, &method, 1, mds,
```

Comment:
seems like there are 2 options:
1. return error code
1. return success, use does mem_query

I think (2) is better because:
1. similar behavior to system mmap
1. error status usually means no side effect happened, but here memory was allocated
1. user which wants to force specific address can use FIXED flag. otherwise, he has to query anyway to discover the real address.